### DISCUSSION TITLE
Does this APP work with Google Watch 2?

### DISCUSSION THREAD

**Discussion by cvancar** [trupici/G-Watch-Wear#40] [Q&A]
Hi guys, Does this APP work with Google Watch 2, please?
Thanks

**Comment by Igor-bubble** [comment]
Galaxy watch 2 is Tizen OS device, not Wear OS. You need to use older app G-Watch. It probably still is available on Samsung play store.

**Reply by psonnera** [reply]
@cvancar sorry for invading this post but I can't reach you: please take down aaps.readthedocs.io
